cluster/kube-push.sh

```diff
@@ -24,10 +24,19 @@ set -o nounset
 set -o pipefail
 
 KUBE_ROOT=$(dirname "${BASH_SOURCE}")/..
+
```

Comment:
`kube-push` is used on more than just e2e clusters - you can't make these changes inline, you'll need to these changes like you did for `e2e-up.sh`